server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/transport/AdmissionControlTransportHandler.java

```diff
@@ -50,16 +54,20 @@ public AdmissionControlTransportHandler(
      */
     @Override
     public void messageReceived(T request, TransportChannel channel, Task task) throws Exception {
-        // intercept all the transport requests here and apply admission control
-        try {
-            // TODO Need to evaluate if we need to apply admission control or not if force Execution is true will update in next PR.
-            this.admissionControlService.applyTransportAdmissionControl(this.action);
-        } catch (final OpenSearchRejectedExecutionException openSearchRejectedExecutionException) {
-            log.warn(openSearchRejectedExecutionException.getMessage());
```

Comment:
there will be 2 warning messages per rejection.
We have warning message here as well
https://github.com/opensearch-project/OpenSearch/pull/10887/files#:~:text=%22CpuBasedAdmissionController%20rejected%20the%20request%20as%20the%20current%20CPU%20%22